Makefile

```diff
@@ -0,0 +1,26 @@
+all: build
+
+FLAGS =
+ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+REGISTRY = gcr.io/google_containers
+TAG = v0.2.0
```

Comment:
Wdyt about moving this into a file? I can see this easily being forgotten when creating a new release...